rawtodigi_alpaka.h

```diff
@@ -0,0 +1,19 @@
+#ifndef RAWTODIGI_ALPAKA_H
+#define RAWTODIGI_ALPAKA_H
+
+#include "pixelgpudetails.h"
+#include "input.h"
+#include "output.h"
+
+#include "alpakaConfig.h"
+
+namespace ALPAKA_ARCHITECTURE{
+
+  namespace Alpaka {
```

Comment:
the namespace here are swapped with respect to `rawtodigi_alpakaAll.cc` ?

Reply:
Yes, I forgot to swap the namespaces in the previous version. I'll fix that!